cli/src/main/java/com/box/l10n/mojito/cli/command/PushCommand.java

```diff
@@ -65,6 +74,12 @@
     @Parameter(names = {"--branch-createdby", "-bc"}, arity = 1, required = false, description = "username of text unit author")
     String branchCreatedBy;
 
+    @Parameter(names = {"--git-diff", "-gd"}, arity = 1, required = false, description = "only processing git diff files")
+    Boolean gitDiff = false;
+
+    @Parameter(names = {"--git-regex", "-gr"}, arity = 1, required = false, description = "only processing git diff files if diff change matches regex")
```

Comment:
Need to update this since the change `--git-diff-ignore` / `gdi`, description: "Regular expression applied to the diff. If matches the diff is ignored"

Reply:
yeah, right!